### PR TITLE
Update respond-file-alerts.md

### PR DESCRIPTION
**Pull request by msbemba**
Added Defender for plan 1 to the applies to as Plan 1 allows file indicators - Verified with the overview document/reference -https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/defender-endpoint-plan-1?view=o365-worldwide#manual-response-actions

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7936

File: microsoft-365/security/defender-endpoint/respond-file-alerts.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ ms.technology: mde
 
 
 **Applies to:**
+-[Microsoft Defender for Endpoint Plan 1](/microsoft-365/security/defender-endpoint/defender-endpoint-plan-1)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/respond-file-alerts.md]:
```suggestion

- [Microsoft Defender for Endpoint Plan 1](/microsoft-365/security/defender-endpoint/defender-endpoint-plan-1)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@mjcaparas Gentle reminder.

**Pull request merged**